### DISCUSSION TITLE
Image in background.

### DISCUSSION THREAD

**Discussion by Kesanov** [emilk/egui#713] [Q&A]
Is it possible to have an image in the background behind classic widgets?

Additionally:
- Does each scaling of an image require new allocation?
- Why does TexAllocator have a free method, when rust doesn't use manual memory management?

**Comment by parasyte** [comment]
I answered your questions on a different thread: https://github.com/emilk/egui/issues/447#issuecomment-917661213 There was also a very similar question asked yesterday: https://github.com/emilk/egui/discussions/710

**Reply by charleschege** [reply]
@emilk @parasyte it would be nice to have an example of this even for #710 in discussions. Background image and background gradients
